packages/react-accordion/Spec.md

```diff
@@ -0,0 +1,369 @@
+# Accordion
+
+## Background
+
+### Definition
+
+This spec defines the default function of an `Accordion` as a vertically stacked set of interactive panels that each contain a title and content snippet.
+
+An accordion is a vertically stacked group of collapsible sections. An accordion is composed of grouped buttons and panels. When a user selects an accordion button, its corresponding panel should switch between 'open' and 'collapsed' states.
+
+Accordions follow many consistent patterns but do allow for some variability in behavior. For example, some accordions only allow one panel to be open at a time, where others may allow multiple or all panels to be open simultaneously. Similarly, many accordions will allow all panels to be simultaneously collapsed, while others may require one panel to be open at all times.
+
+If you are familiar with the disclosure pattern, an accordion will feel very similar. The key distinction is that a disclosure is a standalone component that consists of a single button-panel-group. Because of this, you cannot navigate between different disclosures with a keyboard the same way you can with an accordion.
+
+## Prior art
+
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([research proposal](https://github.com/WICG/open-ui/pull/263))
+
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
+
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
+
+There's no comparison to be done between the two libraries, since v8 hasn't implemented this component.
+
+## Issues
+
+- [#16553](https://github.com/microsoft/fluentui/issues/16553)
+- [#16452](https://github.com/microsoft/fluentui/issues/16452)
+- [#12953](https://github.com/microsoft/fluentui/issues/12953)
+- [#12238](https://github.com/microsoft/fluentui/issues/12238)
+
```

Comment:
Can you please clarify a purpose of this section? From my POV only a first one has some relationship to Accordion...

Reply:
Well. the purpose was to just add all current issues in the current version the Accordion from previous versions. I'll take a look at them!